### PR TITLE
Enable api and update image

### PR DESCRIPTION
**Pull request by blackzlq**
1. Enable api for dlp and nature language api
2. update frontend image sha

### CODE REVIEW

File: applications/rag/frontend/main.tf [hunk 1]
```diff
@@ -15,6 +15,24 @@ data "google_project" "project" {
   project_id = var.project_id
 }
 
+# Enabled the DLP service
+resource "google_project_service" "dlp_api" {
```
Comment (imreddy13) [inline applications/rag/frontend/main.tf:19]: Tested tf apply and tf destroy work as expected I hope?

**Review by imreddy13 — APPROVED** [review]

**Review by imreddy13 — DISMISSED** [review]

File: applications/rag/frontend/main.tf (hunk 1 above)
Comment (blackzlq) [reply applications/rag/frontend/main.tf:19]: The job is using default SA, https://cloud.google.com/build/docs/automating-builds/create-manage-triggers which lack of the permission, I tested on my local. it works

Comment (blackzlq) [reply applications/rag/frontend/main.tf:19]: We need to update the sa for e2e test, this code change is ok

Comment (imreddy13) [reply applications/rag/frontend/main.tf:19]: Does that mean this will be broken for customer projects too?

Comment (blackzlq) [reply applications/rag/frontend/main.tf:19]: Two cases:
1. if customer using terraform, they should use a sa like owner (or we should give a permission list what they need)
2. if they deploy through gcp, then we can create the sa for them with the certain permission, like what we are doing today.

Comment (blackzlq) [reply applications/rag/frontend/main.tf:19]: After I add service admin permission it passed the test

**Review by bjornsen — APPROVED** [review]

**Pull request merged**